### PR TITLE
Add GPU slurm scripts and EKS GPU results for OSU and LAMMPS

### PR DESCRIPTION
**Pull request by milroy**
Several SLURM scripts and data and results for LAMMPS and OSU on the size 16 `p3dn.24xlarge` EKS capacity reservation.

### CODE REVIEW

File: experiments/aws/parallel-cluster/gpu/experiment/README.md [hunk 1]
```diff
@@ -11,18 +11,21 @@
...
+singularity pull docker://ghcr.io/converged-computing/metric-single-node:cpu && \
+singularity pull docker://ghcr.io/converged-computing/metric-amg2023:spack-slim && \
+singularity pull docker://ghcr.io/converged-computing/metric-kripke-gpu:libfabric && \
+singularity pull docker://ghcr.io/converged-computing/metric-laghos:libfabric-gpu && \
+singularity pull docker://ghcr.io/converged-computing/metric-magma:libfabric && \
+singularity pull docker://ghcr.io/converged-computing/metric-lammps-cpu:libfabric && \
+singularity pull docker://ghcr.io/converged-computing/metric-minife:libfabric && \
+singularity pull docker://ghcr.io/converged-computing/metric-mixbench:libfabric-gpu && \
+singularity pull docker://ghcr.io/converged-computing/mt-gemm:libfabric && \
+singularity pull ghcr.io/converged-computing/multi-gpu-models:libfabric && \
```
Comment (vsoch) [inline experiments/aws/parallel-cluster/gpu/experiment/README.md:23]:
```suggestion
singularity pull docker://ghcr.io/converged-computing/multi-gpu-models:libfabric && \
```

File: experiments/aws/parallel-cluster/gpu/experiment/README.md [hunk 2]
```diff
@@ -11,18 +11,21 @@
...
+singularity pull ghcr.io/converged-computing/pytorch-resnet-experiment:libfabric-gpu && \
```
Comment (vsoch) [inline experiments/aws/parallel-cluster/gpu/experiment/README.md:24]:
```suggestion
singularity pull docker://ghcr.io/converged-computing/pytorch-resnet-experiment:libfabric-gpu && \
```

File: experiments/aws/eks/gpu/README.md [hunk 3]
```diff
@@ -736,32 +725,33 @@ And then run as follows.
 oras login ghcr.io --username vsoch
 app=osu
 output=./results/$app
-mkdir -p $output
-
-./flux-run-combinations.sh 8 $app
+mkdir -p $output/size-${size}
```
Comment (vsoch) [inline experiments/aws/eks/gpu/README.md:728]:
```suggestion
mkdir -p $output
```

File: experiments/aws/eks/gpu/README.md [hunk 4]
```diff
@@ -303,38 +303,25 @@ app=lammps-reax
...
-
-time flux run --env CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 -o gpu-affinity=per-task -o cpu-affinity=per-task -N4 -n 32 -g 1 lmp -k on g 8 -sf kk -pk kokkos newton on neigh half -in in.reaxff.hns -v x 64 -v y 64 -v z 32 -in in.reaxff.hns -nocite
-
-time flux run -o gpu-affinity=per-task -o cpu-affinity=per-task -N4 -n 32 -g 1 lmp -k on g 4 -sf kk -pk kokkos newton on neigh half -in in.reaxff.hns -v x 64 -v y 64 -v z 32 -in in.reaxff.hns -nocite
-
 # Size 4
 for i in $(seq 1 5); do     
   echo "Running iteration $i"
-  time flux submit --env CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 --setattr=user.study_id=$app-4-iter-$i -o gpu-affinity=per-task -o cpu-affinity=per-task -N4 -n 32 -g 1 lmp -k on g 4 -sf kk -pk kokkos newton on neigh half -in in.snap.test -var snapdir 2J8_W.SNAP -v x 128 -v y 128 -v z 128 -var nsteps 20000
+  time flux run --env CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 -o cpu-affinity=per-task --setattr=user.study_id=$app-4-iter-$i -N4 -n 32 -g 1 lmp -k on g 8 -sf kk -pk kokkos cuda/aware off newton on neigh half -in in.reaxff.hns -v x 64 -v y 32 -v z 32 -in in.reaxff.hns -nocite
```
Comment (vsoch) [inline experiments/aws/eks/gpu/README.md:309]: :tada:

File: experiments/azure/cyclecloud/gpu/README.md [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+
+#### Nek5000
+
+```bash
+mkdir /shared/nekrs
+cd /shared/nekrs/
+oras pull ghcr.io/converged-computing/metric-nek5000:libfabric-cpu-data
+for i in {1..5}; do sbatch --output=../../data/nekrs/%x-%j-iter-${i}.out --error=../../data/nekrs/%x-%j-iter-${i}.err slurm-nekrs-32n.sh; done
+cd ../../data/nekrs
+oras push ghcr.io/converged-computing/metrics-operator-experiments/performance:azure-cyclecloud-cpu-32node-nekrs nekrs
```
Comment (vsoch) [inline experiments/azure/cyclecloud/gpu/README.md:181]: All of these URIs need to be changed to `azure-cyclecloud-gpu-32` (right now you will overwrite the CPU results)

Comment (milroy) [reply experiments/azure/cyclecloud/gpu/README.md]: Good catch.

**Comment by milroy** [comment]
Thanks for the feedback; I updated the PR @vsoch.

**Review by vsoch — APPROVED** [review]

**Pull request merged**